janusgraph-server/src/test/java/org/janusgraph/graphdb/server/JanusGraphSettingsTest.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 JanusGraph Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.janusgraph.graphdb.server;
+
+import org.apache.tinkerpop.gremlin.server.Settings;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class JanusGraphSettingsTest {
+
+    @Test
+    public void testGrpcServerDefaultValues() throws Exception {
+        JanusGraphSettings settings = JanusGraphSettings.read("src/test/resources/janusgraph-server-without-serializers.yaml");
+
+        assertFalse(settings.grpcServer.enable);
+        assertEquals(10182, settings.grpcServer.port);
+    }
+
+    @Test
+    public void testGrpcServerOverwriteDefaultValues() throws Exception {
+        JanusGraphSettings settings = JanusGraphSettings.read("src/test/resources/janusgraph-server-with-server-configs.yaml");
```

Comment:
`janusgraph-server-with-server-configs.yaml` sounds a bit confusing to me, coz it gives me an illusion that `janusgraph-server-with(out)-serializers.yaml` does not contain server configs. What about `janusgraph-server-with-rpc.yaml`?